csharp-intermediate-topics/TupleAliasInCSharp/TupleAliasInCSharp/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿global using tupleAliasGlobalUsing = (int num1, string str, int num2);
```

Comment:
The tuple alias name should be UpperCamelCase, and we can move this into a separate file called Usings.cs

Reply:
This PR has been updated to reflect recent changes.